### PR TITLE
Add key order readme

### PR DESCRIPTION
**Pull request by jyao1**

### CODE REVIEW

File: doc/key_byte_order.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+|Link_Enc_Key_0[31:24] |Key_DW7_Byte0 |byte[3]:  52   |
+|Link_Enc_Key_0[39:32] |Key_DW6_Byte3 |byte[4]:  05   |
+|Link_Enc_Key_0[47:40] |Key_DW6_Byte2 |byte[5]:  6e   |
+|Link_Enc_Key_0[55:48] |Key_DW6_Byte1 |byte[6]:  02   |
+|Link_Enc_Key_0[63:56] |Key_DW6_Byte0 |byte[7]:  e0   |
+|-                     |-             |-              |
+|Link_Enc_Key_1[7:0]   |Key_DW5_Byte3 |byte[8]:  ef   |
+|Link_Enc_Key_1[15:8]  |Key_DW5_Byte2 |byte[9]:  8b   |
+|Link_Enc_Key_1[23:16] |Key_DW5_Byte1 |byte[10]: 7f   |
+|Link_Enc_Key_1[31:24] |Key_DW5_Byte0 |byte[11]: be   |
```
Comment (mxu9) [inline doc/key_byte_order.md:62]: byte[11] is eb.

File: doc/key_byte_order.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+|Key_Slot_DW3[23:16]    |Key_DW3_Byte1 |byte[18]: b8   |
+|Key_Slot_DW3[15:8]     |Key_DW3_Byte2 |byte[17]: 4e   |
+|Key_Slot_DW3[7:0]      |Key_DW3_Byte3 |byte[16]: 6a   |
+|-                      |-             |-              |
+|Key_Slot_DW4[31:24]    |Key_DW4_Byte0 |byte[15]: d9   |
+|Key_Slot_DW4[23:16]    |Key_DW4_Byte1 |byte[14]: d4   |
+|Key_Slot_DW4[15:8]     |Key_DW4_Byte2 |byte[13]: 39   |
+|Key_Slot_DW4[7:0]      |Key_DW4_Byte3 |byte[12]: 97   |
+|-                      |-             |-              |
+|Key_Slot_DW5[31:24]    |Key_DW5_Byte0 |byte[11]: be   |
```
Comment (mxu9) [inline doc/key_byte_order.md:143]: byte[11] is eb?

File: doc/key_byte_order.md (hunk 1 above)
Comment (jyao1) [reply doc/key_byte_order.md]: thanks. fixed.

File: doc/key_byte_order.md (hunk 2 above)
Comment (jyao1) [reply doc/key_byte_order.md]: thanks. fixed.

**Pull request merged**